api/test/test_helper/fake_android_plugin.gd

```diff
@@ -65,3 +98,34 @@ func getPackLocation(pack_name : String):
 	if pack_name in _asset_pack_location_store:
 		return _asset_pack_location_store[pack_name]
 	return null
+
+func getPackLocations():
+	return _asset_pack_location_store
+
+# -----------------------------------------------------------------------------
+# Simulates the cancel() function in PlayAssetDelivery Android plugin. Iterate
+# through all AssetPackState within _asset_pack_states_store and return the
+# updated states.
+# -----------------------------------------------------------------------------
+func cancel(pack_names : Array):
+	var return_asset_pack_states = {
```

Comment:
Since you use those values here and in clear_asset_pack_states_store you could make a `const var` for `empty_asset_pack_states` and reference that.

Reply:
Refactored into ```const EMPTY_ASSET_PACK_STATES```